scss/sidebar.scss

```diff
@@ -1,9 +1,21 @@
-.sidebar-wrapper {
+.btn.btn-sidebar-toggle:not(.bulk-select, .sidebar-section-header) {
+  border: 1px solid dark-light-diff($secondary, $primary, 20%, -20%) !important; // overrides another important
```

Comment:
Not a blocker, but using dark-light-diff shold ideally be done in a color-definitions file (so it is correctly respecting OS dark mode switching).